components/add_groups_to_team_modal/add_groups_to_team_modal.tsx

```diff
@@ -212,11 +232,12 @@ export default class AddGroupsToTeamModal extends React.PureComponent {
         );
     }
 
-    renderValue(props) {
+    // public for tests
+    public renderValue(props: { data: Value }): string | undefined {
         return props.data.display_name;
     }
```

Comment:
this can be simplified as well:

```suggestion
    public renderValue = (props: { data: Value }): string | undefined =>props.data.display_name;
```